helm/strider/values.yaml

```diff
@@ -93,6 +93,7 @@ ingress:
   strider:
     host: "the strider host name"
     annotations:
+      kubernetes.io/ingress.class: nginx
```

Comment:
this needs to move to the ncats values files, as it is specific to ITRB's enviroment